Week-2/2.1/index.html

```diff
@@ -0,0 +1,71 @@
+                </article>
+                <img src="assests/green-building.jpg" alt="A green building">
+            </section>
+            <section class="container">
+                <article class="right-article">
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quidem minima alias laboriosam veritatis illo suscipit
+                        aut tempora delectus qui, molestiae earum asperiores nisi fugit minus, consectetur perspiciatis, eligendi
+                        obcaecati repellat odit praesentium dolorum! Sunt facere eius reiciendis ipsa deleniti.</p>
+                </article>
+                <article class="right-article">
+                    <h1>Lorem ipsum dolor sit amet.</h1>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquid animi deleniti autem, accusamus consectetur
+                        debitis odit laudantium fugiat! Totam consequatur mollitia dignissimos reprehenderit. Eos omnis ducimus, error
+                        odio, quam illum in libero dolor quod, nostrum sed commodi quisquam iste?</p>
+                </article>
+                <article class="right-article">
+                    <h1 class="bottom-h1">Lorem ipsum dolor sit amet.</h1>
+                    <p class="bottom-p">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Placeat earum dolore, cumque veritatis, nobis odit
+                        porro blanditiis voluptatum suscipit aperiam quisquam dignissimos quas voluptates atque culpa laborum quae? Quae
+                        incidunt vitae provident voluptates unde quisquam, accusamus aut dolor numquam?</p>
+                </article>
+            </section>
+        </section>
+        <section class="form-container">
+            <label> Title: </label>  <label for="mr"> <input id="mr" type="radio" name="title">Mr</label>
+            <label for="mrs"> <input id="mrs" type="radio" name="title">Mrs</label>
+            <label for="miss"> <input id="miss" type="radio" name="title">Miss</label>
+            <br>
+            <label for="firstName">First Name: <input id="firstName" name="firstName" type="text"> </label>
+            <br>
+            <label for="lastName">Last Name: <input id="lastName" name="lastName" type="text"> </label>
+            <br>
+            <label for="email">Email: <input id="email" name="email" type="email"> </label>
+            <br>
+            <label for="phone">Phone Number: <input id="phone" name="phone" type="tel"> </label>
+            <br>
+            <label for="password">Password: <input id="password" name="password" type="password"> </label>
+            <br>
+            <label for="confirm-password">Confirm your password: <input id="confirm-password" name="confirm-password" type="password"> </label>
+            <br>
```

Comment:
Avoid too many HTML elements in DOM, this increases the DOM tree and hence performance.

Use Margin or padding to give some gaps between the elements with style